bench/fixture.ts

```diff
@@ -236,17 +237,21 @@ export class BenchFixture {
         {
           sellToken: tokens.id(i % options.tokens).address,
           buyToken: tokens.id((i + 1) % options.tokens).address,
-          feeAmount: ethers.utils.parseEther("1"),
+          feeAmount,
           validTo: 0xffffffff,
           appData: this.nonce++,
           ...orderSpice,
         },
         traders[i % traders.length],
         signingScheme,
-        {
-          executedAmount: ethers.utils.parseEther("100.0"),
-          feeDiscount,
-        },
+        orderSpice.partiallyFillable
+          ? {
+              executedAmount: ethers.utils.parseEther("100.0"),
+              feeDiscount: feeDiscount.div(2),
```

Comment:
Why `.div(2)`?

Reply:
I'll add a comment, basically, the executed fee is half (because the order is partially filled by exactly half). Since the fee discount is a flat amount, this translates to 100% fee discount.